docs/content/stable/yugabyte-platform/alerts-monitoring/performance-advisor.md

```diff
@@ -153,6 +153,7 @@ If range sharding is used, analyze reads and writes and recreate the table with
 
 ## Limitations
 
+- Currently, performance advisor is partially supported for YCQL-only universes. For details, contact {{% support-platform %}}.
```

Comment:
Don't understand what we're communicating here. Is the solution to turn on YSQL like with ASH, is that what we are referring to? Why contact support? Why don't we say something like

Currently, performance advisor is only partially supported for YCQL-only universes. To access all features, turn on the YSQL API. See [instructions]